### ISSUE TITLE
[BUG] using the outer key keeps showing unauthorized use of the script.

### ISSUE THREAD

**Issue by MatTZer28** [dashingsoft/pyarmor#1102]
Sorry, it's me again. As the [TypeError problem #1057](https://github.com/dashingsoft/pyarmor/issues/1057) has been resolved since last time, I continue to follow section 1.3.5 of the document.

Here is my result :

Firstly, I ran the command 'pyarmor gen --outer foo.py' and it executed correctly.
```
PS D:\Work\pyarmor_demo> pyarmor gen --outer foo.py
INFO     Python 3.11.2
INFO     Pyarmor 8.0.26 (trial), 000000, non-profits
INFO     search inputs ...
INFO     find script foo.py
INFO     find 1 top resources
INFO     start to generate runtime files
INFO     target platforms {'windows.amd64'}
INFO     write dist\pyarmor_runtime_000000\pyarmor_runtime.pyd
INFO     patch runtime file
INFO     generate runtime files OK
INFO     start to obfuscate scripts
INFO     process resource "foo"
INFO     obfuscating file foo.py
INFO     write dist\foo.py
INFO     obfuscate scripts OK
```

Next, I attempted to run the 'foo.py' file in the dist folder. However, it displayed an unauthorized use of script error. Despite this, I assumed that the --outer option was intended to produce this result.
```
PS D:\Work\pyarmor_demo> python dist/foo.py     
Traceback (most recent call last):
  File "<frozen __main__>", line 3, in <module>
RuntimeError: unauthorized use of script (1:10499)
```

Referring to the documentation, I generated a runtime key by executing the command 'pyarmor gen key -e 3', which ran without any issues.
```
PS D:\Work\pyarmor_demo> pyarmor gen key -e 3   
INFO     Python 3.11.2
INFO     Pyarmor 8.0.26 (trial), 000000, non-profits
INFO     start to generate outer runtime key "pyarmor.rkey"
INFO     write dist\pyarmor.rkey
INFO     generate outer runtime key OK
```

Subsequently, I moved the 'pyarmor.rkey' file located in the dist folder to the 'pyarmor_runtime_000000' folder.
```
cp .\dist\pyarmor.rkey .\dist\pyarmor_runtime_000000\
```

Finally, I attempted to run 'foo.py', but it still displayed an unauthorized use of script error.
```
PS D:\Work\pyarmor_demo> python dist/foo.py     
Traceback (most recent call last):
  File "<frozen __main__>", line 3, in <module>
RuntimeError: unauthorized use of script (1:10499)
```

I wonder why the error keeps showing up. Which step did I do wrong?

**Comment by jondy** [comment]
Reproduced, I'll check it and fixed in next release.

**Comment by jondy** [comment]
Fixed in `8.1.dev7`
